tests/main/interfaces-mount-control-nfs/test-snapd-sh/meta/snap.yaml

```diff
@@ -0,0 +1,16 @@
+name: test-snapd-mount-control-nfs
+summary: A snap for testing mount-control with NFS
+version: 1.0
+
+apps:
+    cmd:
```

Comment:
This should be called sh, not cmd.  We have two styles of shell like tests, one that is like cmd and runs $@, effectively, and one that is sh and runs /bin/sh "$@". Those are not the same and we should be clear what is it that we run.